### PR TITLE
Separate msgpack types from serialization details [2/2]

### PR DESCRIPTION
**Pull request by SX91**
This is the second part of https://github.com/TokTok/hs-msgpack-types/pull/1

This change is [Reviewable](https://reviewable.io/reviews/toktok/hs-msgpack/43)

### CODE REVIEW

File: data-msgpack.cabal [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name:                 data-msgpack
-version:              0.0.9
+version:              0.1.0
```
Comment (iphydf) [inline data-msgpack.cabal]: Why is this a minor version bump? Why not 0.0.10?

**Comment by iphydf** [comment]
Please enable the checkbox "Allow edits from maintainers." on the bottom right.

**Comment by SX91** [comment]
@iphydf It's already checked. And I've fixed version to be a patch bump.

**Comment by iphydf** [comment]
Any idea why appveyor is failing?

**Comment by iphydf** [comment]
:lgtm_strong:

---

Review status: 0 of 14 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/hs-msgpack/43#-:-Kljx7-MYlLyJZhV7kI6:b-jbsf8z)*

**Comment by iphydf** [comment]
@SX91 you have push access to this repo, so once it's ready, feel free to merge.

**Comment by iphydf** [comment]
@SX91 turns out, chocolatey is broken. #44 has a fix. Let's release your PR as 0.0.11.

**Comment by SX91** [comment]
So, what about actually merging this?

**Comment by robinlinden** [comment]
:lgtm_strong:

---

Reviewed 13 of 14 files at r1, 1 of 1 files at r5.
Review status: all files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/hs-msgpack/43#-:-Kt6jhYHyvpjNk6CH5QN:b-jbsf8z)*

**Pull request merged**